### DISCUSSION TITLE
Supporting --post-renderer-args would be great!

### DISCUSSION THREAD

**Discussion by fuog** [helmfile/helmfile#1078] [Ideas]
Hi,
i was wondering why helmfile does not support `--post-renderer-args`. From my point of view it would have the advantage that 
- I would not have to write a bash script anymore with a tool like ytt. (seems more clean to me)
- the configuration of the post-renderer itself could be controlled with helmfile itself. so for example `ytt -f- -f file1.yaml -f -file2.yaml` could be conditioned `-f file2.yaml`.

Helm --help:
```
....
      --post-renderer postRendererString           the path to an executable to be used for post rendering. If it exists in $PATH, the binary will be used, otherwise it will try to look for the executable at the given path
      --post-renderer-args postRendererArgsSlice   an argument to the post-renderer (can specify multiple) (default [])
...
```

thanks for your great work on this helm-wrapper :heart:

**Comment by yxxhero** [comment]
@fuog PR is welcome.

**Comment by boekhold** [comment]
It's in the works as PR #1133
